static/index.html

```diff
@@ -19,6 +19,10 @@
   <body>
     <header>
       <h1 class="title">Remote Play</h1>
+      <label class="switch">
```

Comment:
The user sees a switch, but the intent is not clear. 

Something like this will be good:
<img src="https://user-images.githubusercontent.com/34853447/97795042-e8a55500-1bd7-11eb-8e92-376fac2ec7e3.png" width="200" />

Or perhaps a less fancier version (with text/icon) will also do:
<img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTKLblfC-bhgpKhB27uXAkm1FRbsm896cv66w&usqp=CAU" width="200" />